index.html

```diff
@@ -26,27 +24,188 @@ <h1>Hey there. I'm Richard</h1>
          </div>
           <div class="support">
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-  don’t hestiate to contact me.</p>
+          don’t hestiate to contact me.</p>
           </div>
           <div class="social-media">
             <a href="#"><i class="fab fa-twitter social"></i></a>
             <a href="#"><i class="fab fa-linkedin social"></i></a>
             <a href="#"><i class="fab fa-medium-m social"></i></a>
-            <!-- <a href="#"><i class="fab fa-medium social"></i></a> -->
             <a href="#"><i class="fab fa-github social"></i></a>
             <a href="#"><i class="fab fa-angellist social"></i></a>
-
           </div>
-
-          <div class="arrow">
+          <section class="arrow">
             <button  class="arrow-button" ><i class="fas fa-chevron-down fa-2x"></i></button>
-          </div>
+          </section>
         </section>
+   </div>
+   <section class="grid-container">
 
-     <!-- </div> -->
+       <section class="work-intro">
+         <h3>My Recent Works</h3>
+         <hr>
+       </section>
 
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <h4>Multi-Post Stories <br> Gain+Glory</h4>
+            </div>
+            <div class="container-projects">
 
-   </div>
+              <ul class="container-projects-element">
+                 <li>Ruby on rails</li>
+                 <li>CSS</li>
+                 <li>Javascript</li>
+                 <li>html</li>
+              </ul>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <h4>Multi-Post Stories <br> Gain+Glory</h4>
+            </div>
+            <div class="container-projects">
+
+              <ul class="container-projects-element">
+                 <li>Ruby on rails</li>
+                 <li>CSS</li>
+                 <li>Javascript</li>
+                 <li>html</li>
+              </ul>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
```

Comment:
[OPTIONAL]
- There is no image with the name '1234.png'. Please, Kindly fix it to make our project more professional.

Reply:
Noted, and thank you.